wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,291 @@
+<!-- TODO: strange descriptions in many places and unhelpful order if you're trying to see what this game mode is all about. -->
+
+# osu! (mode permainan)
+
+*Untuk kegunaan lain, lihat [osu! (disambiguasi)](/wiki/Disambiguasi/osu!).*
+
+![Gameplay dari osu!](/wiki/shared/osu-gameplay.jpg "osu! Interface")
+
+Mode permainan pertama yang namanya didasarkan dari seri ouendan.
+
+## Apa itu "Ouendan"?
+
+*[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* adalah permainan ritme untuk *Nintendo DS*. Alur permainan ini terdiri dari tiga elemen: mengetuk lingkaran pada layar sentuh, menyeret bola melintasi jalur dan memutar *spinner* dengan sangat cepat. Semua elemen ini diatur waktunya oleh (cover dari) lagu-lagu jepang populer, Ini akan terlihat seperti ini di *DS*:
+
+![Contoh alur permainan dari Osu! Tatakae! Ouendan di Nintendo DS](/wiki/shared/Ouendan.jpg "Gameplay example of Osu! Tatakae! Ouendan in Nintendo DS")
+
+Ceritanya bisa dilihat di layar atas, dan lingkarannya bisa dilihat di layar bawah. Setiap tahap pada dasarnya adalah cerita pribadi tentang seseorang yang bermasalah. Di situlah *Ouendan* (Skuad Penyorak) masuk. Dengan kekuatan magis dari pemandu sorak pria, pemain harus membantu orang yang bermasalah.
+
+## ![](/wiki/shared/mode/osu.png) Alur Permainan
+
+*Tutorial: [new beginnings (Sudah termasuk di dalam osu!)](https://osu.ppy.sh/beatmapsets/1011011).*
+
+Alur permainannya menekankan pada:
+
+- Adaptasi dan reaksi terhadap rangsangan (yaitu, objek yang dapat diketuk muncul di seluruh layar dengan kecepatan kilat).
+- Memahami ketukan lagu dan pola.
+- Kontrol perangkat masukan.
+- Waktu klik pada objek yang muncul di layar.
+- Kemampuan untuk memutar mouse (atau pena untuk tablet) pada *spinner*.
+- Stamina fisik dan mental (terutama lagu jenis maraton yang lebih panjang).
+- Penentuan posisi yang tepat dan koordinasi mata-tangan.
+
+Berikut ini adalah beberapa keterampilan penting untuk menjadi *juara di game osu!*.
+
+Langkah pertama untuk memulai perjalanan adalah mempelajari dasar-dasar (dan teori) dari objek-objek yang dapat dipukul (disebut sebagai *hit objek* mulai sekarang dan seterusnya).
+
+### Pemilihan Lagu
+
+Untuk mengakses mode permainan di osu!, tekan `Ctrl` +` 1` secara bersamaan.
+
+Cara lainnya, klik tombol `Mode` dan pilih` osu! `.
+
+### Dasar-dasar Alur Permainan
+
+#### Area Bermain
+
+![antarmuka osu!](/wiki/shared/osu-gameplay.jpg "osu! Interface")
+
+Bilah kiri atas adalah bilah kesehatan, yang akan menurun dengan kecepatan tetap (tergantung pada kesulitan beatmap), tetapi dapat diisi ulang dengan mengetuk nada pada waktu yang tepat atau memutar *spinner*. Pukulan yang tepat waktunya (300 atau Geki) akan memulihkan kesehatan lebih baik daripada pukulan yang tidak tepat waktunya (50). pukulan yang gagal akan menghilangkan sebagian kesehatan dari bilah kesehatan.
```

Comment:
```suggestion
Bilah kiri atas adalah bilah HP (health bar), yang akan menurun dengan kecepatan tetap (tergantung pada kesulitan beatmap), tetapi dapat diisi ulang dengan mengetuk nada pada waktu yang tepat atau memutar *spinner*. Pukulan yang tepat waktunya (300 atau Geki) akan memulihkan HP ebih baik daripada pukulan yang tidak tepat waktunya (50). pukulan yang gagal akan menghilangkan sebagian HP dari bilah HP.
```

Reply:
```kesehatan``` doesn't really sound it would be fitting

Reply:
i think so, but i was confused about what to replace it with. Thanks for the suggestions, i will recheck it all